### ISSUE TITLE
Inaccuracies in the README

### ISSUE THREAD

**Issue by myrrlyn** [vlang/v#319]
The README states that V was to be released on 2019-06-22T18:00Z but it is 2019-06-22T18:06Z and no release exists

----

edit: it's 18:24Z and a release exists, ty

----

I downloaded the compiler source and IcarusVerilog couldn't process it but there are no other compilers available that can process `.v` files, please fix

**Comment by Bowero** [comment]
Get the pitchforks!

**Comment by xacrimon** [comment]
Get Linus over here. ITS TIME FOR A RANT

**Comment by kianmasri** [comment]
Can I sue for this?

**Comment by qookei** [comment]
Absolutely yes.

**Comment by daniel-j** [comment]
vvvvvvvaporware! /s

**Comment by mintsuki** [comment]
This issue gets worse overtime, pls fix.

**Comment by TheStr3ak5** [comment]
write in C

**Comment by mintsuki** [comment]
I'd like to note a smartass PhD just called V lang a meme

**Comment by xacrimon** [comment]
Woah streaks and mint is here.

**Comment by xacrimon** [comment]
Mr Korona PhD man come over and roast this meme.

**Comment by clmay** [comment]
Is there a livestream I am missing, or something?

**Comment by Restioson** [comment]
With my PHP in HTML I decree this project a meme

**Comment by umitdemirci** [comment]
xd

**Comment by xacrimon** [comment]
Don't forgot your thesis in PDF's rest

**Comment by mintsuki** [comment]
@clmay yes https://discord.gg/28umRq

**Comment by DutchGhost** [comment]
MEME. REVIEW.

**Comment by clmay** [comment]
I mean, not even a tweet??

**Comment by zac-garby** [comment]
It turns out the online playground is a bunch of people manually executing your code 🤔

**Comment by noonien** [comment]
I'm running out of champagne

**Comment by eagle453** [comment]
nique ta mère aiden

**Comment by MinecraftBoy2009** [comment]
does this work on TempleOS?

**Comment by Restioson** [comment]
> Don't forget your thesis in PDF's rest

Don't worry, I've moved on to greater things. I'm aiming for a PDF in PHPs

**Comment by xacrimon** [comment]
Oh, how cool. You will now be automagically accepted to Oxford.

**Comment by eagle453** [comment]
hmmmmmmmmmmmmmmm my big v-lang

**Comment by myrrlyn** [comment]
@zac-garby

> It turns out the online playground is a bunch of people manually executing your code

Oh neat I didn't know it was hosted on Mechanical Turk

**Comment by qookei** [comment]
where my v.c at
please fix
i would like to convert linux to v

**Comment by xacrimon** [comment]
oh man

**Comment by Restioson** [comment]
This is so sad, can we rewrite V in rust

**Comment by Itay2805** [comment]
can we rewrite rust in V

**Comment by Jackojc** [comment]
I'm not going to lie, this is the most excited I've been all week. Waiting for the V release to shit on it.

**Comment by playXE** [comment]
Can we rewrite world in V

**Comment by xacrimon** [comment]
VAlexa play vountry roads. Vust

**Comment by TheStr3ak5** [comment]
lmao

**Comment by qookei** [comment]
ITT V is a scam and we'll never get it

**Comment by UditDey** [comment]
I will fucking kill myself if V doesn't come out right now

**Comment by Bowero** [comment]
Look at all these comments.. I wonder why he does'nt publish it...

**Comment by qookei** [comment]
yeah, he's afraid his scam will come to light

**Comment by zac-garby** [comment]
Honestly, he's probably just finishing off something 🤷‍♂

**Comment by xacrimon** [comment]
He should announce that via twitter then.

**Comment by daniel-j** [comment]
Brb gonna play some VVVVVV.

**Comment by Itay2805** [comment]
when you make such bold claims and doesn't release in time, you probably made a mistake

**Comment by Bowero** [comment]
> Honestly, he's probably just finishing off something

Which is why he should make it open source. Look at all these people willing to help.

**Comment by hyarsan** [comment]
yikes

**Comment by kianmasri** [comment]
I sold my first born child so I could live to June 22nd for V's release. If it gets delayed again I don't know what I'll do.

**Comment by Miaourt** [comment]
*blobcatbongo*

**Comment by playXE** [comment]
_dies_

**Comment by mintsuki** [comment]
btw guys i have asuper epic project which totally exists but it's not open source until may 15th, 94023984 A.D.

**Comment by hyarsan** [comment]
i like how this issue updates live, it's like discord with the reactions

**Comment by Miaourt** [comment]
Hey, nice new chat

**Comment by mdodis** [comment]
I could've gone out for drinks tonight, but honestly this is totally worth it

**Comment by qookei** [comment]
this issue thread is the perfect chat platform tbh

**Comment by mintsuki** [comment]
this ngl

**Comment by xacrimon** [comment]
Who even needs discord anymore.

**Comment by Restioson** [comment]
I have created a truly marvellous language, which this git repository is too small to contain

**Comment by juliancoffee** [comment]
олды тут?

**Comment by MinecraftBoy2009** [comment]
*commits suicide*, nothing to live for now.

**Comment by mintsuki** [comment]
quintessential github shitposting

**Comment by Itay2805** [comment]
so true, github issues are better than expected

**Comment by kianmasri** [comment]
github is my preferred social network

**Comment by zakkor** [comment]
whats up guys

**Comment by playXE** [comment]
@juliancoffee да

**Comment by Miaourt** [comment]
Gonna make a XEP for XMPP over GIthub Issues

**Comment by binkiklou** [comment]
OPEN YOUR EYES SHEOPLE V IS AN ILLUSION

**Comment by Restioson** [comment]
I'm abandoning Vertex

**Comment by CoenSchutte** [comment]
"V is written in V, and you can build it in 0.4 seconds." 
No use if it takes 50 years to become available

**Comment by zac-garby** [comment]
lol

**Comment by xacrimon** [comment]
No rest. I need something to use ccl

**Comment by sijad** [comment]
convert git to v

**Comment by vikrrrr** [comment]
It's released!

**Comment by xacrimon** [comment]
You cant do that @Restioson

**Comment by coolreader18** [comment]
I really want to post `# Right now V can only be installed in "~/code", this is a temporary limitation` to r/pcj on reddit, also maybe this thread.

**Comment by Alkouille** [comment]
How do I exit vim ?

**Comment by SleepyRoy** [comment]
The code is open sourced!

**Comment by binkiklou** [comment]
wait really

**Comment by kianmasri** [comment]
Pog

**Comment by Restioson** [comment]
@xacrimon add a PR to V to use it

**Comment by binkiklou** [comment]
This thread is out of control

**Comment by coolreader18** [comment]
hallelujah

**Comment by Bowero** [comment]
ITS AVAILABLE

**Comment by hyarsan** [comment]
it's out it's out we can celebrate

**Comment by Miaourt** [comment]
Woaaaaaw

**Comment by qookei** [comment]
pogchamp

**Comment by binkiklou** [comment]
Our prayers have been answered

**Comment by coolreader18** [comment]
and now there's no readme also

**Comment by zakkor** [comment]
IT'S OUT

**Comment by Restioson** [comment]
This can be closed, since the README was removed

**Comment by zac-garby** [comment]
Wait a second, how can we build it without a binary?

**Comment by hello2dj** [comment]
IT'S OUT

**Comment by qookei** [comment]
never
dont close

**Comment by hyarsan** [comment]
no readme is a bummer

**Comment by hyarsan** [comment]
who needs a v lang discord chat when we have this

**Comment by coolreader18** [comment]
oh and he force-pushed, so we can't even look back to see what the install instructions were

**Comment by zac-garby** [comment]
F it doesn't build

**Comment by binkiklou** [comment]
How are we going to build this, if we don't have binaries

**Comment by Bowero** [comment]
README now included

**Comment by zac-garby** [comment]
I suppose he'll add a binary as a release?

**Comment by Bowero** [comment]
@medvednikov, ignore the mean things we sad: we love you

**Comment by zac-garby** [comment]
Ah, in fact you can download a C version of it from https://vlang.io/v.c apparently

**Comment by nico-abram** [comment]
When i tried running curl -O https://vlang.io/v.c  on windows i got an html page

**Comment by mintsuki** [comment]
i'd like to point out a new issue: V lang actually came out

**Comment by daniel-j** [comment]
it's because it doesn't exist at v.c

**Comment by coolreader18** [comment]
well where is it then???

**Comment by Restioson** [comment]
We're still waiting on v.c

**Comment by zakkor** [comment]
does anyone have a V compiler?

**Comment by coolreader18** [comment]
Oh yeah there were pre-release binaries right?

**Comment by zac-garby** [comment]
It was released for a little while actually wasn't it. Someone's bound to!

**Comment by binkiklou** [comment]
All we have is a Mac OS binary

**Comment by nico-abram** [comment]
This is beautiful https://github.com/vlang/v/blob/master/rand/rand.v#L12

**Comment by zac-garby** [comment]
https://www.xkcd.com/221/

**Comment by UditDey** [comment]
Uses fucking shell commands to do stuff what a retard

**Comment by zac-garby** [comment]
I bet v.c is somewhere on the vlang.io webspace. I wonder if we can find it

**Comment by binkiklou** [comment]
There is that binary https://archive.org/details/v_macos but it;s old I think and for mac too.

**Comment by zac-garby** [comment]
I'll try and built it with that

**Comment by nico-abram** [comment]
Cross compile it for us windows plebs

**Comment by zac-garby** [comment]
That's assuming it works at all 🤔

**Comment by coolreader18** [comment]
And Linux please!

**Comment by daniel-j** [comment]
system calls for rmdir and curl, lmao. Vaporware.

**Comment by nico-abram** [comment]
Fastest json parser in the world written in v https://github.com/vlang/v/blob/master/json/json_primitives.v

**Comment by zac-garby** [comment]
`main.v` won't build, "unknown type `Table`"... Probably outdated binary?

**Comment by coolreader18** [comment]
v.c is available!

**Comment by coolreader18** [comment]
i got a segfault trying to build the compiler with vc

**Comment by zac-garby** [comment]
IM IN THE V REPL :))

**Comment by zac-garby** [comment]
but nothing works, it seems to be constructing filepaths wrong because:

```
>>> println("hello world!")
V panic: "/Users/zacgarby//code/v//builtin/array.v" doesnt exist
```

**Comment by daniel-j** [comment]
I got segfault when I tried `println(1+1)`

**Comment by CryZe** [comment]
compiler segfaults for me

**Comment by zac-garby** [comment]
so you have to put the v you download into ~/code

**Comment by zac-garby** [comment]
but for some reason it can't find stdio.h

**Comment by nico-abram** [comment]
I cant compile https://i.imgur.com/idyfWT8.png

**Comment by daniel-j** [comment]
![Screenshot_2019-06-22_20-38-00](https://user-images.githubusercontent.com/1264014/59967692-f8198600-952d-11e9-8421-6fb72ed1a707.png)
great!

**Comment by xacrimon** [comment]
Does this lad even understand basic computer stuff? It seems a lot like he doesnt!

**Comment by zac-garby** [comment]
<img width="260" alt="Screenshot 2019-06-22 at 19 42 05" src="https://user-images.githubusercontent.com/19708658/59967708-d4ead880-9525-11e9-9cc8-3b7ae7279d8f.png">

i did it!!

**Comment by zac-garby** [comment]
i had to compile with clang instead of whatever it was using by default, and i had to put the v directory in ~/code, and i had to manually compile /var/tmp/vlang0.0.12/vrepl.c with clang.

**Comment by hello2dj** [comment]
![image](https://user-images.githubusercontent.com/10645383/59967725-ee5d4600-9560-11e9-88ed-ac11952c524a.png)

**Comment by nico-abram** [comment]
Try random numbers

**Comment by zac-garby** [comment]
<img width="222" alt="Screenshot 2019-06-22 at 19 46 18" src="https://user-images.githubusercontent.com/19708658/59967737-6c502b80-9526-11e9-9ad6-68c932003222.png">

so the repl is actually not a repl

**Comment by zac-garby** [comment]
but how do you import things btw?

**Comment by nico-abram** [comment]
you don't, duh

**Comment by zac-garby** [comment]
```
######
##
####
##
##
```

**Comment by RUSshy** [comment]
look at these Rust bots, they are all mad
Go ask your daddy Crapzilla to feed you some of their

**Comment by RUSshy** [comment]
look at these Rust bots, they are all mad, daddy Mozilla aka Crapzilla didn't feed u enough of cum ?

**Comment by K900** [comment]
> tfw your language does not compile and already makes rust bots mad

**Comment by xacrimon** [comment]
@RUSshy Are you retarded? This is not how it works. Vlang is a clusterfuck. End of discussion. The dev is also deleting valid issues.

**Comment by nico-abram** [comment]
Someone's salty LOL

**Comment by coolreader18** [comment]
https://redd.it/c3t3wz

**Comment by xacrimon** [comment]
This is some huge brain shit my lads. We are going to see a lot of nice articles about this.

**Comment by K900** [comment]
I kinda feel sorry for the guy tbh. Feels like he's not even doing this intentionally...

**Comment by daniel-j** [comment]
I hope that now that this is out in the open, some people aregoing to help. Because the promises of V sounds... promising.

**Comment by zac-garby** [comment]
Yeah but come on, he shouldn't have been accepting all that money.

**Comment by coolreader18** [comment]
Yeah, @medvednikov, we're just memeing, we don't mean anything against you or the work you've put into vlang.

**Comment by RUSshy** [comment]
rust will sink, just like firefox

**Comment by medvednikov** [comment]
The readme was old, that's what was causing the issues.

**Comment by UditDey** [comment]
Does V have a wiki yet? Can we put all this stuff on it?

**Comment by zac-garby** [comment]
It has a website

**Comment by CryZe** [comment]
@medvednikov the new README doesn't solve all the compilation problems though.

**Comment by nico-abram** [comment]
@RUSshy But I'm reading this on firefox

**Comment by betseg** [comment]
hi yes i'm a rust bot daddy mozilla giv cum

**Comment by NotNite** [comment]
rust bot reporting in

**Comment by zatherz** [comment]
SO MALD LULW

**Comment by carverh** [comment]
THIS IS A RIP OFF OF https://github.com/mintsuki/storm!!

**Comment by mTvare6** [comment]
> i like how this issue updates live, it's like discord with the reactions

dead chat xd
